site/src/pages/components/toolbar.explainer.mdx

```diff
@@ -0,0 +1,141 @@
+---
+menu: Active Proposals
+name: Toolbar Elements (Explainer)
+path: /components/toolbar.explainer
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- Authors: [@lukewarlow](https://github.com/lukewarlow)
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Introduction](#introduction)
+- [New `<toolbar>` element](#new-toolbar-element)
+- [General questions](#general-questions)
+- [Examples in code](#examples-in-code)
+- [Reading List](#reading-list)
+- [Issues / Discussions](#issues--discussions)
+
+## Introduction
+
+A toolbar is a common user interface pattern used to provide quick access to a group of commands, represented using controls such as [press buttons](/components/press-button.explainer), [menus](/components/menu.explainer) and other such controls.
+
+This proposal introduces a new `<toolbar>` element. The `<toolbar>` will:
+- Represent a grouping of controls (e.g. buttons, select boxes, press buttons).
+- Provide the correct implicit ARIA role and keyboard interaction model out of the box.
+- Allow web developers to create consistent, accessible toolbars without re-implementing ARIA patterns.
+
+Use cases:
+- The formatting toolbar in a document editor (bold, italic, underline, font family).
+- The main command toolbar in a drawing or photo editing app.
+- A simple toolbar with a search field and filter buttons in a catalogue.
+
+## New `<toolbar>` element
+
+`<toolbar>`
+
+- Represents a **group of controls**.
+- Provides implicit `role="toolbar"`.
+- Toolbars are displayed horizontally by default.
+- Provides correct keyboard navigation behaviour.
+
+## General questions
+
+### What should be the keyboard behavior?
+
+This element will implicitly provide the same behaviour as a [toolbar focusgroup](/components/scoped-focusgroup.explainer#supported-behaviors).
+
+- It will contain a single tab-stop.
+- Arrow-keys can be used to navigate between focusable children.
+- Home keys can be used to jump to the beginning or end of the toolbar.
+
+### How can users customize the UI of this toolbar element with CSS?
+
+We propose this element should not support the CSS appearance property and just have a "base" appearance by default like the `<dialog>` element.
+
+**Questions**
+
+* `<toolbar>` should display children horizontally, but support using CSS writing-mode.
+    * Can have vertical case by:
+        * Set CSS `writing-mode: vertical-lr` on the `<toolbar>`
+        * Should this affect arrow-key directionality?
+* Does `<toolbar>` support separators?
+    * If so, we should support `<hr>` for this.
+* Toolbars generally should be labelled, it is a MUST if there are multiple on a page. Should we support this via HTML,
+  or still rely on aria?
+    * Could use `<legend>` to do this?
+
```

Comment:
I'd like to throw in another general question on how to handle keyboard behavior when authors throw in weird / complex children (e.g. a select that already has keyboard behavior, or nested toolbars, etc. etc.)

Reply:
Added that in and linked to the focusgroup relevant issue.